### PR TITLE
CSS tweaks + proposed content changes

### PR DESCRIPTION
**Pull request by sinak**
- CSS fixes to make buttons and labels a bit clearer.
- Cleaned up homepage on text to make it more "accurate." This is based
  on feedback from previous campaigns - people (particularly in places like
  HN and reddit) are really sensitive to anything that might construed as
  overly "activisty" or sensationalist. It's a tough line to walk though, and I
  may have gone too far in the opposite direction. feel free to revert some
  of my text changes if you think they make the content weaker.

### CODE REVIEW

**Comment by sinak** [comment]
About to board a flight - haven't reviewed anything past the first two screens, but will do so either later tonight or in the morning.

File: dmca-comments.css [hunk 1]
```diff
@@ -0,0 +1 @@
...px;margin:36px 0}.action-nav.top-action{margin-top:-25px}.accordion-row{position:relative;margin:12px 0}.accordion-container{cursor:pointer}.accordion-title{font-weight:bold}.accordion-description{margin-left:56px}.exemption-label{color:#d0021b;font-size:16px;text-transform:uppercase;margin-bottom:4px;font-weight:bold}.accordion-slide{display:none;margin-left:56px}.accordion-slide textarea{width:100%}.links{margin-left:56px}.accordion-link{margin-left:6px}.letter,.exemptions,.contact{display:none}.error-message,.has-error{color:#d0021b}.success-page{text-align:center}.success-page .page p{margin:16px 0}.success-page .page img{width:100%}.success{font-size:48px;text-transform:uppercase;text-align:center}.footer{text-align:center;margin:48px 0}.logos{list-style-type:none;margin:48px auto;padding:0;overflow:hidden;width:946px}@media only screen and (max-width: 599px){.logos{width:100%}}.logos li{display:inline-block;float:left;margin:0 24px}@media only screen and (max-width: 599px){.logos li{margin:24px !important}}.logos li:first-of-type{margin-left:0}.logos li:last-of-type{margin-right:0}.logos img{max-width:150px}@media only screen and (max-width: 599px){.logos img{max-width:100%}}
```
Comment (timothyasp) [inline dmca-comments.css:1]: We don't need to commit this css.  The build and deploy process converts less -> css and minifies.

File: dmca-comments.less [hunk 2]
```diff
@@ -118,6 +118,8 @@ input[type=checkbox] + label:before {
 
 label {
    font-size: 22px;
+   cursor: pointer;
+   cursor: hand;
```
Comment (timothyasp) [inline dmca-comments.less:122]: Double?

**Comment by timothyasp** [comment]
Closing this and applying these changes seperately from this pull.  Thanks @sinak

File: dmca-comments.less (hunk 2 above)
Comment (sinak) [inline dmca-comments.less:122]: Yeah you need both of these to be backwards compatible with old versions of IE.

**Pull request closed without merging**